app/models/issue.rb

```diff
@@ -31,7 +32,7 @@
 class Issue < ApplicationRecord
   include CsvExportable
 
-  belongs_to :school
+  belongs_to :issueable, polymorphic: true #, optional: true
```

Comment:
does `#, optional: true` need to be removed or uncommented here?

Reply:
I left it like this as it might be optional in the future... but I'll just make a note of the syntax elsewhere instead :-)